maldoca/base/file.cc

```diff
@@ -223,6 +229,16 @@ absl::Status GetContents(const std::string& path, std::string* contents) {
     }
     absl::StrAppend(contents, absl::string_view(buf, ret));
   }
+  std::string content_tmp = *contents;
```

Comment:
GetContent is a general utility and this introduces unexpected behavior for general use. I think you should add a parameter "decode_as_base64" with default false so the caller controls if decoding is needed.  You can move this naming logic to a wrapper function in maldoca/maldoca/base/testing/test_utils.h to set decode_as_base64 based on names just for testing.

Reply:
also you only need declare content_tmp if you need to decode so should only be done in the if(decode) clause.  Then use std::move to move the data not copy.

Reply:
added utility function in test_utils and a default decode_as_base64 to GetContents.